server/app.js

```diff
@@ -1,5 +1,18 @@
+require('env2')('.env');
 const express = require('express');
+const cookieParser = require('cookie-parser');
+const { join } = require('path');
+const router = require('./routes');
 
 const app = express();
 
+app.set('PORT', process.env.PORT || 4000);
+app.disable('x-powered-by');
+app.use(cookieParser());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(express.static(join(__dirname, '..', 'client', 'public')));
```

Comment:
we are going to run our client-side into different ways, In production, our files will be in `build` folder, and in the development, we don't need to send those files as we will run `npm start`, 
So we need to serve static files conditionally based on the environment we run